### PR TITLE
increase agent memory for azure as well

### PR DESCRIPTION
**Pull request by kvalliyurnatt**
We see e2e tests failing on aks with agent in OOM, increase memory in e2e tests for agent in aks tests

### CODE REVIEW

**Comment by kvalliyurnatt** [comment]
buildkite test this -f p=aks,TESTS_MATCH=TestFleetMode

**Review by barkbay — APPROVED** [review]

File: test/e2e/test/agent/builder.go [hunk 1]
```diff
@@ -132,8 +132,8 @@ func NewBuilder(name string) Builder {
 		WithLabel(run.TestNameLabel, name).
 		WithDaemonSet()
 
-	if test.Ctx().OcpCluster {
-		// Agent requires more resources on OpenShift clusters. One hypothesis is that
+	if test.Ctx().OcpCluster || test.Ctx().AksCluster {
+		// Agent requires more resources on OpenShift and aks clusters. One hypothesis is that
```
Comment (pebrc) [inline test/e2e/test/agent/builder.go:136]:
```suggestion
		// Agent requires more resources on OpenShift and AKS clusters. One hypothesis is that
```

**Comment by thbkrkr** [comment]
buildkite test this -f p=tanzu,TESTS_MATCH=TestFleetMode

**Comment by kvalliyurnatt** [comment]
buildkite test this -f p=tanzu,TESTS_MATCH=TestFleetMode

**Comment by kvalliyurnatt** [comment]
> buildkite test this -f p=tanzu,TESTS_MATCH=TestFleetMode
> 
> Agent e2e tests seem also flaky on tanzu. Maybe we should do the same, even if the one triggered by this has passed?

Added in tanzu as well @thbkrkr, please take another look.

**Pull request merged**